src/ProviderBlinktrade.js

```diff
@@ -0,0 +1,42 @@
+const Lang = imports.lang;
+const Local = imports.misc.extensionUtils.getCurrentExtension();
+const BaseProvider = Local.imports.BaseProvider;
+
+const _invalidExchangeError = () =>
+    new Error("use_average !== true and no exchange defined");
+
+const Api = new Lang.Class({
+  Name: "Blinktrade.Api",
+  Extends: BaseProvider.Api,
+
+  apiName: "Blinktrade (Bitcambio)",
+
+  apiDocs: [
+    ["API Docs", "https://blinktrade.com/docs/"]
+  ],
+
+  interval: 30, // unclear, should be safe
+
+  getUrl({ base, quote }) {
+    // log("DEBUG: Blinktrade: getUrl(): base="+base+", quote="+quote)
+
+    if (base !== undefined) {
```

Comment:
I think you meant `base === undefined`

please remove it though to not set surprising defaults, the user should just enter `BTC`/`BRL` himself